src/frontends/ir/tests/CMakeLists.txt

```diff
@@ -4,11 +4,18 @@
 
 set(TARGET_NAME ov_ir_frontend_tests)
 
+list(APPEND IR_TESTS_DEPENDENCIES openvino_ir_frontend openvino_intel_cpu_plugin)
+if (NOT UNIX OR NOT ENABLE_INTEL_CPU)
+    list(REMOVE_ITEM IR_TESTS_DEPENDENCIES openvino_intel_cpu_plugin)
+    set(EXCLUDED_TESTS ${CMAKE_CURRENT_SOURCE_DIR}/frontend_test_mmap.cpp)
+endif()
+
 ov_add_test_target(
         NAME ${TARGET_NAME}
         ROOT ${CMAKE_CURRENT_SOURCE_DIR}
+        EXCLUDED_SOURCE_PATHS ${EXCLUDED_TESTS}
```

Comment:
You can use ifdef to exclude some tests if macro is not defined

Reply:
Thanks for suggestion! Anyway, CPU dependency is removed